features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/MicrogatewayLabels/AddEditMGLabel.jsx

```diff
@@ -86,6 +87,23 @@ function AddEditMGLabel(props) {
     const onChange = (e) => {
         dispatch({ field: e.target.name, value: e.target.value });
     };
+
+    const handleHostValidation = (hostName) => {
+        const schema = Joi.string().uri().empty();
+        const validationError = schema.validate(hostName).error;
+
+        if (validationError) {
+            const errorType = validationError.details[0].type;
+            if (errorType === 'any.empty') {
+                return 'Host is empty';
+            }
+            if (errorType === 'string.uri') {
```

Comment:
we can do this in an elseif() right? Do we need 2 if conditions?

Reply:
Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/8535/commits/12e3962cb2b86c5983390a9d137c7184595b0cd5